Makefile

```diff
@@ -357,7 +357,7 @@ tests/%.run-truffle: tests/%
 	tests/truffle/runtest.sh $(dir $@)
 
 %.run-openzep:
-	tests/run-openzep.sh $*
+	timeout 1800 tests/run-openzep.sh $*
```

Comment:
Don't put the `timeout` here, because this is system dependent how fast it will run.

Instead, just make sure there is a global timeout on the Jenkins stage this runs in.